scripts/install/opt/seagate/cortx/motr/common/m0_utils_common.sh

```diff
@@ -494,3 +521,85 @@ get_m0repair_utils_cmd()
         cmd_trigger="$M0_REPAIR_UTILS -O $op -t $SERVICE_TYPE -C ${LOCAL_NID}${cli_ep} $IOS_EP "
         echo "$cmd_trigger"
 }
+
+get_dix_pool_fid()
+{
+        local conf_cg="/tmp/conf.cg"
+
+        generate_conf_cg "$conf_cg"
+        if [ ! -f "$conf_cg" ]; then
+                echo "$conf_cg not present, can't get dix pool fid"
+                exit
+        fi
+
+        # Get pool version and then pool fid from conf.cg
+        pver=$(grep imeta_pver "$conf_cg" | grep -o -P '(?<=imeta_pver=).*?(?= )')
+        pool=$(grep "$pver" "$conf_cg" | grep -v imeta_pver | grep "pool-" | grep -o -P '(?<=pool-).*?(?= )')
+
+        pool="0x6f00000000000001:$pool"
+        rm -f "$conf_cg"
+        echo "$pool"
+}
+
+get_fids_list()
+{
+        local HCTL_STATUS_FILE="/tmp/hctl_status.log"
+        hctl status > "$HCTL_STATUS_FILE"
+
+        host=$(hostname)
+        if [ "$SERVICE_TYPE" -eq 1 ]; then # DIX
+                POOL_FID=$(get_dix_pool_fid)
+        else
+                POOL_FID=$(grep -A2 'Data pool:' "$HCTL_STATUS_FILE" | awk '{print $1}' | tail -n 1)
+        fi
+        PROFILE_FID=$(grep -A2 'Profile:' "$HCTL_STATUS_FILE" | awk '{print $1}' | tail -n 1)
+        HA_ENDPOINT_ADDR=$(grep -A 50 "$host" "$HCTL_STATUS_FILE" | grep -m 1 hax | awk '{print $4}')
+
+        rm -f "$HCTL_STATUS_FILE"
+}
+
+_create_fid_list()
```

Comment:
Please document what the "$1" is.
What does this function do?
What is the result?

Reply:
Done.